client/src/main/java/org/apache/celeborn/client/read/DfsPartitionReader.java

```diff
@@ -98,6 +98,17 @@ public DfsPartitionReader(
           ShuffleClient.getHdfsFs(conf).open(new Path(location.getStorageInfo().getFilePath()));
       chunkOffsets.addAll(getChunkOffsetsFromUnsortedIndex(conf, location));
     }
+    StringBuilder builder = new StringBuilder();
+    for (Long offset : chunkOffsets) {
+      builder.append(offset + " ");
+    }
+    logger.debug(
```

Comment:
use logger.isDebugEnabled, if true then put concat chunkOffsets and logger.debug into it's logic scope, but why not print offset list directly?

Reply:
OK.